test/unit/files_for_processing_test.rb

```diff
@@ -43,6 +43,30 @@ def setup
       assert_equal files, files.uniq
     end
 
+    test "fetch with custom paths without honoring package boundaries includes only include glob in custom paths including nested package files" do
+      files = ::Packwerk::FilesForProcessing.fetch(paths: ["."], configuration: @configuration,
+honor_package_boundaries: false)
```

Comment:
```suggestion
        honor_package_boundaries: false)
```

Reply:
Mhh, that's strange, don't we have a rubocop that should catch these things?